### ISSUE TITLE
Haven't mention where do I need paste filesync.mapping

### ISSUE THREAD

**Issue by kirankotari** [CatStarwind/FileSync#8]
Is that in settings.json file? or a new file?

**Comment by GilliamOperator** [comment]
It is in settings.json. You can get to it from the UI too:
![image](https://user-images.githubusercontent.com/1597258/83221070-a3ad5c00-a129-11ea-8d9f-856d02edad80.png)

Example:
![image](https://user-images.githubusercontent.com/1597258/83221178-f38c2300-a129-11ea-8e86-ed085c091d87.png)

**Comment by kirankotari** [comment]
You meant "under the project folder .vscode -> settings.json" right. I did that it didn't work. 
vs code version 1.45.1 -> when I activate File Sync I am always getting No mapping found for "souce_path" all the time. I checked couple of things to work this out but couldn’t ended with solution. 

So, I have created a new Extension called [RunCommandOnSave](https://github.com/kirankotari/RunCommandOnSave) <- User has a privilege to run any command he likes on file save. 

![image](https://user-images.githubusercontent.com/1794046/83331467-4265c400-a2b4-11ea-90a8-d1f748777fdd.png)

**Comment by GilliamOperator** [comment]
No, this is a VSCode (User) setting not a project setting. You should be able to bring up the User settings.json with <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>p</kbd> and typing ``settings``.

![image](https://user-images.githubusercontent.com/1597258/83338549-47a61d00-a27a-11ea-9cf5-ec904b2186bb.png)

It might be an idea to support project settings in the future.

**Comment by cyboh** [comment]
Hi, by putting the the mapping in the global settings (Windows %APPDATA%\Code\User\settings.json) does worked; but may I know am I able to put the settings in my project's folder itself? I tried to create a .vscode folder inside my project folder and created and settings.json inside it. But somehow it doesn't read the settings inside my project folder.

I read it from here to do it in folder:
https://supunkavinda.blog/vscode-editing-settings-json#folder
